images/linux/ubuntu1604.json

```diff
@@ -129,6 +129,14 @@
             ],
             "execute_command": "sudo sh -c '{{ .Vars }} {{ .Path }}'"
         },
+        {
+            "type": "shell",
+            "inline": [
+                "mkdir -p /etc/skel/.config/configstore",
+                "echo 'export XDG_CONFIG_HOME=$HOME/.config' | tee -a /etc/skel/.bashrc"
```

Comment:
This line doesn't work without using external provisionator, the same issue was at https://github.com/actions/virtual-environments/blob/master/images/linux/scripts/installers/dotnetcore-sdk.sh.

Reply:
I've tested it on deployed VM and it works fine. Are there any changes made by the provisioner that breaks it?

Reply:
I've got your idea, but let's leave it for self-hosted agents.